Makefile

```diff
@@ -122,7 +129,7 @@ EXT = so
 VERSION_EXT = $(EXT).$(API_MAJOR)
 AR_EXT = a
 $(LIBNAME)_LDFLAGS += -Wl,-Bsymbolic-functions,-soname,lib$(LIBNAME).$(VERSION_EXT)
-UNICORN_CFLAGS += -fvisibility=hidden
+# UNICORN_CFLAGS += -fvisibility=hidden
```

Comment:
why did you comment out this line? i am not sure this is false positive, but Travis reported an issue for this PR.